jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CfnIntrinsicFunction.kt

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation
+
+/**
+ * [Official Docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html)
+ */
+enum class CfnIntrinsicFunction(val id: String) {
```

Comment:
I know we have to make this file but I really wish we didn't have to

Reply:
Ya, you haven't seen the best part yet 😄